samza-core/src/main/java/org/apache/samza/context/SamzaContainerContextImpl.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.samza.context;
+
+public class SamzaContainerContextImpl implements SamzaContainerContext {
```

Comment:
I assume that this will be removed together with SamzaContainerContext?

Reply:
Yep